### DISCUSSION TITLE
Managing multiple activity diagrams at the top level

### DISCUSSION THREAD

**Discussion by Jeremaster999** [gaphor/gaphor#2840] [Q&A]
Hi,
I´m just getting started with Gaphor and MBSE in general in the scope of my master thesis.
If I understand the workflow with Gaphor correctly, I am supposed to, at the concept level, put different use cases of the system in the use cases diagram pregenerated in Gaphor. For each of these use cases, I think it would make sense to create a seperate activity diagram to model the wanted behaviour on a concept level. However, I can´t create new activity diagrams on the same level as the pregenerated one. They always get placed in an automatically generated "New Activity" folder which makes it so I can´t place these activity diagrams on the top level like the pregenerated diagram is. Am I missing something? Do I get the workflow wrong, am I supposed to put the activity flow in only one diagram but next to each other?
Help would be very much appreciated, as I generally like the UI etc. but I can´t get the workflow going!

**Comment by danyeaw** [comment]
Hi @Jeremaster999, thanks for your patience with a response, and thanks for the question!

Since we created the starting SysML template, it looks like we enhanced some of our Activity Diagram functionality. The new functionality is that when you create a new Activity Diagram, it automatically creates an Activity, and nests the Activity Diagram under it. You can then double-click on the activity and add it to the diagram to make it the border of the activity inside the diagram.

Finally, you can create new actions inside the activity, which would look something like this:

![image](https://github.com/gaphor/gaphor/assets/10014976/3889e0da-ec08-44e9-9927-2520b5290877)

Would you be interested in updating the SysML starting template with this new style for us? We love contributions!
